pkg/actions/start.go

```diff
@@ -13,14 +13,25 @@ package actions
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/eclipse/codewind-installer/pkg/utils"
+	logr "github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
 // StartCommand : start the codewind containers
 func StartCommand(c *cli.Context, dockerComposeFile string, healthEndpoint string) {
-	status := utils.CheckContainerStatus()
+	printAsJSON := c.GlobalBool("json")
+	status, err := utils.CheckContainerStatus()
+	if err != nil {
+		if printAsJSON {
+			fmt.Println(err.Error())
+		} else {
+			logr.Error(err.Desc)
+		}
+		os.Exit(1)
```

Comment:
This is done so frequently it would be nice to see it pulled out to a utility function.
Something like:
```suggestion
if err != nil {
		printAsJSON := c.GlobalBool("json")
		if printAsJSON {
			fmt.Println(err.Error())
		} else {
			logr.Error(err.Desc)
		}
		os.Exit(1)
}
```
There will still be a few places that don't use this but there are also follow up PR's like: https://github.com/eclipse/codewind-installer/pull/316
that would benefit from this.